docs/user/sbt.rst

```diff
@@ -16,17 +16,17 @@ This generates the following files:
 
 * ``project/plugins.sbt`` to add a plugin dependency::
 
-    addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.4.0-M2")
+    addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.4.0")
 
 * ``project/build.properties`` to specify the sbt version::
 
-    sbt.version = 1.4.1
+    sbt.version = 1.4.5
```

Comment:
btw https://eed3si9n.com/sbt-1.4.6

Reply:
Should we really change this? 1.4.6 does not seem to be officially out. Maybe write "or later" somewhere?

Reply:
The link I posted above is about as official announcement as they come. I guess that with the current cadence of releasing sbt (which is good!), somebody didn't update the scala-sbt.org/download.html website. It's a manual process and easy to forget.

But this is really just a minuscule point :wink: if people use 1.4.1 or 1.4.5, Scala Steward will come and upgrade it anyway (and Mergify will merge it) to what will be at that time 1.4.7 or later :smile:

Reply:
I find announcements of sbt versions are, to be kind, hard to find.
The [sbt repository](https://github.com/sbt/sbt) lists 1.4.6 as the latest
(now 27 days old) version.

Reply:
Yes, better say "or later" in the text as we might miss an upgrade.

Reply:
Scala Native itself uses `1.4.6`.